language/stdlib/modules/libra_system.mvir

```diff
@@ -253,12 +297,36 @@ module LibraSystem {
         // TODO: support adding and removing validators. Eventually, we will do this by computing
         // the new validator set from a larger list of candidate validators sorted by stake.
         validator_set_ref = borrow_global_mut<ValidatorSet>(0x1D8);
+        additions_ref = &mut copy(validator_set_ref).additions;
         vector_ref = &mut copy(validator_set_ref).validators;
 
+        // (1) Check for additions
+        i = 0;
+        len = Vector.length<address>(freeze(copy(additions_ref)));
+        if (copy(len) > 0) {
```

Comment:
What do you think about making the dependency of `changed` on `len` here more explicit? e.g. 

```rust 
changed = copy(len) > 0;
if (copy(changed)) {
   ....
}
```

Reply:
I will add a comment for this.